controller_interface/package.xml

```diff
@@ -14,14 +16,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>roscpp</build_depend> 
-  <build_depend>hardware_interface</build_depend> 
-  <build_depend>pluginlib</build_depend> 
-  <run_depend>roscpp</run_depend> 
-  <run_depend>hardware_interface</run_depend> 
-  <run_depend>pluginlib</run_depend> 
-
-  <export>
-    <cpp cflags="-I${prefix}/include"/>
-  </export>
+  <depend>roscpp</depend>
```

Comment:
Given how big is the PR, I think this could be done on another PR if you prefer.